src/components/ChatLog.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+
+const ChatLog = (props) => {
+    const ChatLogs = props.entries.map((message) => {
```

Comment:
The convention is to use capital letter variable names for React components. This is technically an array of React elements, so it would be more aligned with conventions to use a lower case letter variable name. However, I am personally not fond of this pattern (even though it is shown in Learn) and have not found it to be common in practice.